vale/styles/vocab.txt

```diff
@@ -22,7 +24,6 @@ Explainshell
 Firewalled
 Flexbox
 Fugit
-GitHub
```

Comment:
GitHub is in [the default dictionary](https://github.com/errata-ai/en_US-web/), so we don't need this. 🎉